dev-db/db-je/db-je-3.3.75-r1.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+JAVA_PKG_IUSE="doc source"
+
+inherit java-pkg-2 java-ant-2
+
+DESCRIPTION="A high performance, transactional storage engine written entirely in Java"
+HOMEPAGE="https://www.oracle.com/database/berkeley-db/java-edition.html"
+SRC_URI="https://download.oracle.com/berkeley-db/${P/db-/}.tar.gz"
+
+LICENSE="Sleepycat BSD"
+SLOT="3.3"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=virtual/jdk-1.8:*
+	dev-java/ant-core:0"
+RDEPEND=">=virtual/jre-1.8:*"
+
+S=${WORKDIR}/${P/db-/}
+
+EANT_DOC_TARGET="javadoc-all"
+EANT_EXTRA_ARGS="-Dnotest=true"
+
+PATCHES=(
+	"${FILESDIR}"/${P}-optional-test.patch
+	# allow using source="1.8" target="1.8"
+	"${FILESDIR}"/db-je-3.3.75-build.xml.patch
```

Comment:
i had the impression that our build.xml rewrite code also updates the source and target versions in the build.xml file. this does not work automagically?

Reply:
> i had the impression that our build.xml rewrite code also updates the source and target versions in the build.xml file. this does not work automagically?

I was not aware of rewrite code, so did not check this.  At least `ant/compile.xml` needs manual update.

Reply:
see https://github.com/gentoo/gentoo/blob/master/eclass/java-ant-2.eclass#L102 and related, you're interested in the bsfix stuff. you might try to set `JAVA_PKG_BSFIX_NAME=( build.xml compile.xml )` to handle `compile.xml` as well.

Reply:
Applied `JAVA_ANT_REWRITE_CLASSPATH`, but doesn't seem to work for `ant/compile.xml`.
And I prefer keeping it limited to `:1.8` reserving (wip) the push to java-11 for [db-je-7.5.11](https://www.oracle.com/database/technologies/related/berkeleydb-downloads.html).

Reply:
to scan and process other files than `build.xml` you have to use the `JAVA_PKG_BSFIX_NAME` variable.